_posts/national/united-kingdom/united-kingdom.Rmd

```diff
@@ -56,63 +56,106 @@ We calculate Rt as the average of how many new infections arise from one infecte
-    ggplot2::geom_line(ggplot2::aes(y = median),
-              alpha = 0.9, size = 1) +
     ggplot2::geom_hline(yintercept = 1, linetype = 2) +
-    ggplot2::coord_cartesian(ylim = c(NA, 1.5 * max(region_summary[["median"]]))) +
+    ggplot2::labs(y = "R(t)", x = NULL) +
     ggplot2::scale_color_manual(values = colours) +
     ggplot2::scale_fill_manual(values = colours) +
-    ggplot2::scale_x_date(date_breaks = "1 week", date_labels = "%b %d") +
+    ggplot2::scale_x_date(date_breaks = "2 weeks", date_labels = "%b %d") +
     cowplot::theme_cowplot() +
-    ggplot2::labs(y = "R(t)", x = NULL, subtitle = region_name) +
-    ggplot2::theme(legend.position = "bottom",
-          axis.text.x = ggplot2::element_text(angle = 45, hjust = 1)) +
+    ggplot2::theme(axis.text.x = ggplot2::element_text(angle = 90, hjust = 1)) +
     ggplot2::guides(colour = ggplot2::guide_legend(override.aes = list(alpha = 1)))
+  }
+  
+    else {
+  p +
+    ggplot2::geom_point(ggplot2::aes(y = count)) +
+    ggplot2::geom_line(ggplot2::aes(y = count_ma)) +
+    ggplot2::labs(y = "Log count", x = NULL, title = region_name) +
+    ggplot2::scale_color_manual(values = colours) +
+    ggplot2::scale_fill_manual(values = colours) +
+    ggplot2::scale_x_date(date_breaks = "2 weeks", date_labels = "%b %d") +
+    cowplot::theme_cowplot() +
+    ggplot2::theme(axis.text.x = ggplot2::element_blank()) +
+    ggplot2::guides(colour = "none", fill = "none") +
+    ggplot2::scale_y_log10()
+  }
 }
+
+region_plot_data <- purrr::map(region_names,
+                             ~ plot_fn(region_name = .x, rt = FALSE))
 region_plot_rt <- purrr::map(region_names,
-                             ~ plot_rt_fn(region_name = .x))
+                             ~ plot_fn(region_name = .x, rt = TRUE))
+
 # # Plot selected regions:
-region_plot_rt[[1]] +
-  region_plot_rt[[2]] +
-  region_plot_rt[[3]] +
-  region_plot_rt[[4]] +
-  patchwork::plot_layout(ncol = 2, guides = "collect") &ggplot2::theme(legend.position = "bottom")
+((region_plot_data[[1]] / region_plot_rt[[1]]) | 
+  (region_plot_data[[2]] / region_plot_rt[[2]])) /
+  ((region_plot_data[[3]] / region_plot_rt[[3]]) | 
+  (region_plot_data[[4]] / region_plot_rt[[4]])) +
+  patchwork::plot_layout(guides = "collect") &
+  ggplot2::theme(legend.position = "bottom")
+
 ```
-*Figure 2: Estimates of Rt (median, with 50% (darker shade) and 90% (lightest shade) credible interval), derived from data sources including all test-positive cases, hospital admissions, and deaths with a positive test in the previous 28 days for the devolved authorities of the United Kingdom.*
+*Figure 2: Raw data counts with 7-day moving average; and estimates of Rt (median, with 20% (darker shade) and 50% (lightest shade) credible interval), derived from data sources including all test-positive cases, hospital admissions, and deaths with a positive test in the previous 28 days for the devolved authorities of the United Kingdom.*
```

Comment:
Sorry my fault for not being clear. It would ideally be 20%, 50% and 90% to match the other plots :)